packages/components/modules/messages/web/GroupChatCreate/index.tsx

```diff
@@ -129,36 +128,23 @@ const CreateGroup: FC<CreateGroupProps> = ({
   const isCreateButtonDisabled = !isValid || !isDirty || isMutationInFlight
 
   return (
-    <>
-      <Box>
-        <HeaderContainer>
-          <IconButton onClick={onBackButtonClicked} aria-label="cancel group creation">
-            <CloseIcon sx={{ fontSize: '24px' }} />
-          </IconButton>
-          <Typography component="span" variant="subtitle2" sx={{ textAlign: 'center' }}>
-            New Group
-          </Typography>
-          <IconButton
-            aria-label="Create group"
-            disabled={isCreateButtonDisabled}
-            onClick={() => {
-              onSubmit()
-            }}
-          >
-            <CheckMarkIcon sx={{ fontSize: '24px' }} />
-          </IconButton>
-        </HeaderContainer>
-        <EditGroupTitleAndImage
-          control={control}
-          FORM_VALUE={FORM_VALUE}
-          handleRemoveImage={handleRemoveImage}
-          imageError={getFieldState(FORM_VALUE.image).error}
-          isMutationInFlight={isMutationInFlight}
-          setValue={setValue}
-          trigger={trigger}
-          watch={watch}
-        />
-      </Box>
+    <Box>
+      <Header
```

Comment:
we could also make this header a default component and allow for having a custom header.
also will be good to have the header props here so that the group create can change the header icons